aiohttp/web_fileresponse.py

```diff
@@ -48,6 +48,19 @@
     {ext: mimetypes.encodings_map[ext] for ext in (".br", ".gz")}
 )
 
+FALLBACK_CONTENT_TYPE = "application/octet-stream"
+
+# https://en.wikipedia.org/wiki/List_of_archive_formats#Compression_only
+ENCODING_CONTENT_TYPES = MappingProxyType(
+    {
+        "gzip": "application/gzip",
+        "br": "application/x-brotli",
+        "bzip2": "application/x-bzip2",
+        "compress": "application/x-compress",
```

Comment:
`mime_types` uses `Z` instead of `compress` https://github.com/python/cpython/blob/0ec761a96a80f4b84b98aad5e784e4ac8f385c3f/Lib/mimetypes.py#L419

Reply:
This maps the encoding to the type, not the extension.